tee-worker/identity/client-sdk/packages/client-sdk/src/lib/requests/remove-accounts.request.ts

```diff
@@ -0,0 +1,123 @@
+import { hexToU8a } from '@polkadot/util';
+
+import type { ApiPromise } from '@polkadot/api';
+import type {
+  LitentryIdentity,
+  TrustedCallResult,
+  WorkerRpcReturnValue,
+} from '@litentry/parachain-api';
+
+import { enclave } from '../enclave';
+import { codecToString } from '../util/codec-to-string';
+import { createPayloadToSign } from '../util/create-payload-to-sign';
+import { createTrustedCallType } from '../type-creators/trusted-call';
+import { createRequestType } from '../type-creators/request';
+
+import type { JsonRpcRequest } from '../util/types';
+import { getEnclaveNonce } from './get-enclave-nonce';
+
+/**
+ * Removes accounts from the Litentry Parachain.
+ *
+ * @returns {Promise<Object>} - A promise that resolves to an object containing the payload to sign (if applicable) and a send function.
+ * @returns {string} [payloadToSign] - The payload to sign if who is not an email identity.
+ * @returns {Function} send - A function to send the request to the Enclave.
+ * @returns {Promise<Object>} send.args - The arguments required to send the request.
+ * @returns {string} send.args.authentication - The authentication string. If who is
+ * an email identity, this is the email verification code. If who is not an email identity, this is the
+ * signed payload.
+ */
+export async function remove_accounts(
+  /** Litentry Parachain API instance from Polkadot.js */
+  api: ApiPromise,
+  data: {
+    /** Member accounts for removing */
+    memberAccounts: Array<`0x${string}`>;
```

Comment:
These should be an array of the identities to remove. Here is the definition: https://github.com/litentry/litentry-parachain/blob/fd5326b0dd3bc7e7753bfdf3dc7e725d03ab867a/tee-worker/identity/app-libs/stf/src/trusted_call.rs#L149

Reply:
fixed now